connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -703,6 +703,19 @@ public Config describeTopicConfig(String topic) {
         return result;
     }
 
+    protected Map<TopicPartition, Long> retryEndOffsets(Set<TopicPartition> partitions, Duration timeoutDuration, long retryBackoffMs) {
```

Comment:
Why is this protected when `endOffsets(...)` is public? This class is the Connect runtime module, and the class is not part of the public API for Connect.